### ISSUE TITLE
Docker update fails with `ApplyLayer exit status 1 stdout:  stderr: failed to Lchown`

### ISSUE THREAD

**Issue by TheZoker** [n8n-io/n8n#7905]
**Describe the bug**
I just tried to update to the latest `next` version via docker. And this error happened during the `docker compose up` command:
```
failed to register layer: ApplyLayer exit status 1 stdout:  stderr: failed to Lchown "/usr/local/lib/node_modules/n8n/node_modules/@tensorflow/tfjs-node/lib/napi-v8/tfjs_binding.node" for UID 1193602, GID 89939 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /usr/local/lib/node_modules/n8n/node_modules/@tensorflow/tfjs-node/lib/napi-v8/tfjs_binding.node: invalid argument
```

I removed all the containers, images etc. and tried it again but still the same issue. 

**To Reproduce**

1. Use the following docker-compose file:
```
version: '3.8'
services:
  n8n:
    image: n8nio/n8n:next
    container_name: n8n
    restart: always
    ports:
      - 80:5678/tcp
    volumes:
      - ~/files/n8n/data:/home/node/.n8n
      - ~/files/n8n/files:/files
    environment:
      - N8N_USER_MANAGEMENT_DISABLED=false
      - N8N_EDITOR_BASE_URL=https://n8n.domain.com
      - WEBHOOK_URL=https://n8n.domain.com
      - GENERIC_TIMEZONE=Europe/Berlin
```
Run `docker compose up`

**Expected behavior**
Work

**Environment (please complete the following information):**

- OS: Alpine OS 3.16
- n8n Version: 1.19.4
- Node.js Version: ?
- Database system: SQLite
- Operation mode: main

**Additional context**

**Comment by Joffcom** [comment]
Hey @TheZoker,

Just to rule out anything odd on the hardware side, Are you running this one an ARMv7 powered device?

**Comment by TheZoker** [comment]
Hey :)

No it's a x86_64 (i5-9500T)

**Comment by Joffcom** [comment]
Hey @TheZoker,

Can you also try to prune your docker cache and see if that helps

**Comment by TheZoker** [comment]
I deleted the docker cache, but that did not help. Hmm the upgrade was from the "last" next version to the current one. I'm not sure which version that was.
I just switched from the `:next` version to the `:latest` and it works without any issue. 
When I switch back to the `:next` version, there's the issue again.

I also attached a video recording showing the issue. Maybe that helps. The issue happens in the extraction stage of the layer `c973aa845984`.

https://github.com/n8n-io/n8n/assets/1368405/ecd1a7df-22f4-42b0-861a-b3922a7448ee
